mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+### New features
+
+* Mountpoint now automatically adjusts its prefetcher read window size based on available memory. This prevents Mountpoint from consuming all memory in the system resulting in out-of-memory issues. ([#1013](https://github.com/awslabs/mountpoint-s3/pull/1013))
```

Comment:
I'd update the language, since the scope of the issue was related to read patterns. 

```suggestion
* Mountpoint now automatically adjusts its prefetcher read window size based on available system memory. This prevents Mountpoint from consuming all available system memory in cases where a large number of read file handles are open concurrently. ([#1013](https://github.com/awslabs/mountpoint-s3/pull/1013))
```